src/Controllers/ElementalAreaController.php

```diff
@@ -456,10 +456,10 @@ private function createValidationExceptionWithNamespaces(
             $messageCast = $message['messageCast'] ?? ValidationResult::CAST_TEXT;
             if ($messageFieldName) {
                 $fieldName = sprintf(EditFormFactory::FIELD_NAMESPACE_TEMPLATE, $element->ID, $messageFieldName);
-                $params = [$fieldName, $messageText, $messageType, null, $messageCast];
+                $params = [$fieldName, $messageText, $messageType, '', $messageCast];
                 $validationResultWithNameSpaces->addFieldError(...$params);
             } else {
-                $validationResultWithNameSpaces->addError($messageText, $messageType, null, $messageCast);
+                $validationResultWithNameSpaces->addError($messageText, $messageType, '', $messageCast);
```

Comment:
This doesn't seem relevant? Looks like https://github.com/silverstripe/silverstripe-elemental/pull/1311 accidentally snuck into this PR?

Reply:
Probably makes sense to block this PR until the other one is merged, then rebase this PR and these changes should vanish